src/entities/join/weighted/helpers.ts

```diff
@@ -0,0 +1,64 @@
+import { JoinInput, JoinKind, PoolState } from '..';
+import { Address } from '../../../types';
+
+export function getJoinParameters({
+    poolId,
+    assets,
+    sender,
+    recipient,
+    maxAmountsIn,
+    userData,
+}: {
+    poolId: Address;
+    assets: readonly Address[];
+    sender: Address;
+    recipient: Address;
+    maxAmountsIn: readonly bigint[];
+    userData: Address;
+}) {
+    const joinPoolRequest = {
+        assets, // with BPT
+        maxAmountsIn, // with BPT
+        userData, // wihtout BPT
+        fromInternalBalance: false,
```

Comment:
I think we should make this part of the input parameters so the user can choose?